bindings/Cargo.toml

```diff
@@ -14,7 +14,7 @@ authors = ["Pravega Community"]
 classifier = ["Development Status :: 5 - Production/Stable", "License :: OSI Approved :: Apache Software License",
     "Programming Language :: Rust", "Programming Language :: Python :: 3.8", "Programming Language :: Python :: 3.9",]
 requires-python = ">= 3.8"
-project-url = ["homepage = https://pravega.github.io/pravega-client-rust/"]
+project-url = {"homepage" = "https://pravega.github.io/pravega-client-rust/"}
```

Comment:
This change prevents the working of maturin version configured in the workflow

Reply:
Nice catch! Some CI is using 0.8.3 while others are using 0.9.0+.